### ISSUE TITLE
[BUG] Inconsisitent output from `npm pkg get`

### ISSUE THREAD

**Issue by RimashMohomed** [npm/cli#6705]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

output of the `npm pkg get` command with single key is different than the command with multiple key.

When the command executed with one key, Output is as below  ie: **npm pkg get name -w worksapceA** 
````
 "@myScope/workspaceA"
{
  "@myScope/workspaceA": "@myScope/workspaceA"
}
````

while the command executed with two keys, Output is as below ie: **npm pkg get name version -w workspaceA**

````
 {
  "name": "@myScope/workspaceA",
  "version": "2.0.53"
}
{
  "@myScope/workspaceA": {
    "name": "@myScope/workspaceA",
    "version": "2.0.53"
  }
}
````

### Expected Behavior

Below is expected regardless of the key count provided to the command.

with one key
````
{
  "@myScope/workspaceA": {
    "name": "@myScope/workspaceA"
}
````

while two keys

````
{
  "@myScope/workspaceA": {
    "name": "@myScope/workspaceA",
    "version": "2.0.53"
  }
}
````

### Steps To Reproduce

In Windows subsystem for Linux 22.04 with `npm` version 9 run `npm pkg get name -w workspaceA` which returns the output inconsistent with the command `npm pkg get name version -w workspaceA`

### Environment

- npm: `9.6.7`
- Node.js: `18.17.0`
- OS Name: `Ubuntu ( 22.04, jammy) on Microsoft Windows 11 Home Single Language`
- System Model Name:  `HP ProBook 440 G8 Notebook PC`
- npm config:
```ini
;  "project" config from /mnt/d/<path to project>/.npmrc

@<variant name>::registry = "https://<company name>.jfrog.io/artifactory/api/npm/npm-local/"
@<company name>:registry = "https://yaala.jfrog.io/artifactory/api/npm/npm-local/"
//<company name>.jfrog.io/artifactory/api/npm/npm-local/:_password = (protected)
//<company name>.jfrog.io/artifactory/api/npm/npm-local/:always-auth = true
//<company name>.jfrog.io/artifactory/api/npm/npm-local/:email = "ci@<company name>.com"
//<company name>.jfrog.io/artifactory/api/npm/npm-local/:username = "cipull"

; node bin location = /<working dir>/.nvm/versions/node/v18.17.0/bin/node
; node version = v18.17.0
; npm local prefix = /<working dir>/<project>
; npm version = 9.6.7
; cwd = <working dir>/<project>
; HOME =  /<working dir>
; 
```

**Comment by Shakeskeyboarde** [comment]
Same problem. Still present in NPM `v10.1.0`.

**Comment by Mitsuo-Yoshida** [comment]
I was redirected here from the node repo, where I opened an issue:
https://github.com/nodejs/node/issues/49931
For me found the bug when i switched the node version
When i updated to any version from `18.14.0` onwards the bug happened to me
I saw that the npm version switched from `8.19.3` to `9.3.1` between 18.13.0 and 18.14.0

**Comment by lukekarrys** [comment]
This was fixed by https://github.com/npm/cli/pull/6977 and released in `npm@10.2.4`
